### PR TITLE
angular-cookies - added options + minor improvements

### PR DESCRIPTION
**Pull request by stephenlautier**

### CODE REVIEW

**Comment by vvakame** [comment]
*angularjs/angular-cookies.d.ts*

to authors(@diegovilar @aciccarello). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

File: angularjs/angular-cookies.d.ts [hunk 1]
```diff
@@ -17,6 +17,30 @@ declare module "angular-cookies" {
 declare module angular.cookies {
 
     /**
+    * Cookies options
+    * see https://docs.angularjs.org/api/ngCookies/provider/$cookiesProvider#defaults
+    */
+    interface ICookiesOptions {
+        /**
+        *  The cookie will be available only for this path and its sub-paths. By default, this would be the URL that appears in your base tag.
```
Comment (aciccarello) [inline angularjs/angular-cookies.d.ts]: I'd prefer these comments be wrapped at 80 or 100 chars.

**Comment by aciccarello** [comment]
:+1: Look good. thanks @stephenlautier

**Comment by stephenlautier** [comment]
@aciccarello done the comments wrapping as suggested. welcome :)

**Comment by vvakame** [comment]
@aciccarello thanks!
@stephenlautier thanks mate!

**Pull request merged**